moveit_core/CMakeLists.txt

```diff
@@ -55,6 +56,10 @@ COMPONENTS
   urdf
   visualization_msgs
   xmlrpcpp
+  tesseract_core
+  tesseract_collision
+  tesseract_msgs
+  tesseract_ros
```

Comment:
@ommmid and I are concerned about this... eventually there will be no dependencies on Tesseract, correct? This is just WIP?

Reply:
You're right, this is still a WIP. The plan is to initially depend on tesseract and see how that interface can be wrapped by the MoveIt collision interface. Then, we'll port over each piece that we need.

Reply:
I'm also not convinced that wrapping another interface is the best approach since the interfaces seem to make different assumptions about collision world and attached objects.